assets/models/system/codellama-7b-instruct-hf/description.md

```diff
@@ -61,24 +68,22 @@ Batch |<a href="https://aka.ms/azureml-infer-batch-sdk-text-generation" target="
 
 ### **Sample input**
 ```json
-{
-  "input_data": {
-      "input_string": ["I believe the meaning of life is"],
-      "parameters":{   
-              "top_p": 0.9,
-              "temperature": 0.6,
-              "max_gen_len": 96,
-              "do_sample": true
-      }
-  }
-}
+{"input_data": {
+    "input_string": ["Develop a Python function to sort a list of integers in ascending order"], "parameters": {"top_p":0.9,
+        "temperature":0.6,
+        "do_sample":true, 
+        "max_new_tokens": 120,
+        "return_full_text":false
+        }
+    }
+} 
 ```
 
 ### **Sample output**
 ```json
 [
-    {
-        "0": "I believe the meaning of life is to learn to love.\\nI believe in a world of compassion, a world where love rules.\\nI believe in a world where people care for one another.\\nI believe in a world where people help each other.\\nI believe in a world where people are kind to each other.\\nI believe in a world where people are happy.\\nI believe in a world where people are peaceful.\\nI believe in a world where people are loving."
-    }
+  {
+    "0": ".\n\n# In[ ]:\n\n\ndef sort_list(my_list):\n    return sorted(my_list)\n\n\n# In[ ]:\n\n\nsort_list([5, 2, 6, 1, 3])\n\n\n# In[ ]:\n\n\ndef sort_list(my_list):\n    return sorted(my_list, reverse=True)\n\n\n# In[ ]:\n\n\nsort_list([5, 2, 6, 1, 3])"
```

Comment:
@skasturi could you confirm if this is okay? Though the results code is okay but it also has py shell `In []`

Reply:
using appropriate output. Resolving this